amy/workshops/util.py

```diff
@@ -681,14 +681,15 @@ def fetch_workshop_metadata(event_url, timeout=5):
     return metadata
 
 
-class WrongWorkshopURL(ValueError):
+class WrongWorkshopURL(Exception):
     """Raised when we fall back to reading metadata from event's YAML front
     matter, which requires a link to GitHub raw hosted file, but we can't get
-    that link because provided URL doesn't match Event.WEBSITE_REGEX
+    that link because provided URL doesn't match Event.WEBSITE_REGEX or Event.REPO_REGEX
     (see `generate_url_to_event_index` below)."""
 
-    def __str__(self):
-        return "Event's URL doesn't match Github website or repo format."
+    def __init__(self, msg: str):
```

Comment:
I believe what you're hoping for is to just show the message of the error. This is achieved without altering the class:
```py
>>> class MyException(Exception):
...    pass
... 
>>> error = MyException("Hi")
>>> str(error)
'Hi'
```

Reply:
Oh nevermind me then... looks like this was what you were trying to fix. Hmmm....